src/libPMacc/include/math/vector/Vector.hpp

```diff
@@ -676,7 +678,7 @@ typename T_Navigator,
 template <typename, int> class T_Storage
 >
 HDINLINE Vector<T_Type, T_Dim>
-operator/(const Vector<T_Type, T_Dim, T_Accessor, T_Navigator, T_Storage>& lhs, const T_Type& rhs)
+operator/(const Vector<T_Type, T_Dim, T_Accessor, T_Navigator, T_Storage>& lhs, typename boost::call_traits<T_Type>::param_type rhs)
```

Comment:
some here `typename Vector<T_Type, T_Dim, T_Accessor, T_Navigator, T_Storage>::type` for `T_Type`